app/web/pages/search.tsx

```diff
@@ -11,7 +12,17 @@ export const getStaticProps: GetStaticProps = translationStaticProps([
 ]);
 
 export default function SearchPage() {
-  return <SearchPageComponent />;
+  const router = useRouter();
+
+  const location = router.query.location || "";
+  const bbox = router.query.bbox || [0, 0, 0, 0];
+
+  return (
+    <SearchPageComponent
+      locationName={location as string}
+      bbox={bbox as [number, number, number, number]}
```

Comment:
Same here add the type to the original variable, in this case you want:

```typescript
  const location: string = router.query.location || "";
  const bbox : number[] = router.query.bbox || [0, 0, 0, 0];

  return (
    <SearchPageComponent
      locationName={location}
      bbox={bbox}
      />
   ```

Reply:
but about the `type Coordinates = [number, number, number, number];` totally good idea, not sure how I didn't though about that before :)

Reply:
Try this, looks like someone already wrote a function to solve this issue.

Add this import:

`import { parsedQueryToSearchFilters } from "utils/searchFilters";`

Then adjust the component like so:

``` typescript
export default function SearchPage() {
  const router = useRouter();
  const query = parsedQueryToSearchFilters(router.query);
  const location = query.location;
  const bbox: Coordinates = query.bbox || [390, 82, -173, -66];

  return <SearchPageComponent locationName={location} bbox={bbox} />;
}
```

Reply:
yes I know about the `parsedQueryToSearchFilters` the thing is, I removed it here in this PR, because this PR it's a refactor to set up the groundwork for the future map, (because the code was really hard to understand) 

and since this PR it's already to big my plan it's to implement the `parsedQueryToSearchFilters` in other PR :)